src/types.ts

```diff
@@ -21,36 +23,32 @@ export type DataInput = {
 }
 
 export type ButtonInput = {
-  type: SpecialInputTypes
+  type: SpecialInputTypes.BUTTON
```

Comment:
I'm happy you sorted out this, I had gotten errors with enforcing type to Button, Monitor, etc.